tests/test_agent_manager.py

```diff
@@ -29,32 +27,48 @@ def model():
     return "gpt-3.5-turbo"
 
 
-@requires_api_key("OPENAI_API_KEY")
-def test_create_agent(agent_manager, task, prompt, model):
+def test_create_agent(request, mocker, agent_manager, task, prompt, model):
```

Comment:
In this file do 

```
@pytest.fixture(autouse=True)
def mock_create_chat_completion(mocker):
    mock_create_chat_completion = mocker.patch(
        "autogpt.agent.agent_manager.create_chat_completion",
        wraps=create_chat_completion,
    )
    mock_create_chat_completion.return_value = "irrelevant"
    return mock_create_chat_completion
```
And you should be able to remove the patch from every test.